src/Components/ModelViewerModePicker/ModelViewerModePicker.tsx

```diff
@@ -120,100 +117,70 @@ const ModelViewerModePicker: React.FC<ModelViewerModePickerProps> = (props) => {
-                        </h4>
-                        <div className={styles.colorPicker}>
-                            <SwatchColorPicker
-                                disabled={
-                                    selectedObjectStyle ===
-                                    ViewerObjectStyle.Default
-                                }
-                                cellHeight={32}
-                                cellWidth={32}
-                                columnCount={colors.length}
-                                selectedId={selectedObjectColor}
-                                cellShape={'circle'}
-                                colorCells={colors}
-                                onChange={updateObjectColor}
-                                getColorGridCellStyles={(props) => {
-                                    if (props.disabled) {
-                                        return {
-                                            colorCell: {
-                                                opacity: '0.1'
-                                            },
-                                            svg: null
-                                        };
-                                    }
-                                }}
-                            />
-                        </div>
-                        <h4 className={styles.subHeading}>
-                            {t('modelViewerModePicker.background')}
-                        </h4>
-                        <div className={styles.colorPicker}>
-                            <SwatchColorPicker
-                                cellHeight={32}
-                                cellWidth={32}
-                                columnCount={backgrounds.length}
-                                selectedId={selectedSceneBackground}
-                                cellShape={'circle'}
-                                colorCells={backgrounds}
-                                onChange={updateBackgroundColor}
-                            />
-                        </div>
-                    </div>
-                </FocusTrapCallout>
-            )}
-        </div>
+                <h4 className={classNames.subHeader}>
+                    {t('modelViewerModePicker.objectColors')}
+                </h4>
+                <div className={classNames.colorPicker}>
+                    <SwatchColorPicker
+                        disabled={
+                            selectedObjectStyle === ViewerObjectStyle.Default
+                        }
+                        cellHeight={32}
+                        cellWidth={32}
+                        columnCount={colors.length}
+                        selectedId={selectedObjectColor}
+                        cellShape={'circle'}
+                        colorCells={colors}
+                        onChange={updateObjectColor}
+                        getColorGridCellStyles={(props) => {
+                            if (props.disabled) {
```

Comment:
should we move it to method?

Reply:
I don't think there's any harm in leaving it. I actually didn't add it so am just going to leave this one here since it's short.